helm/values.yaml

```diff
@@ -0,0 +1,97 @@
+# Default values for selfcare-uservice-party-management.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: selc
+
+replicaCount: 1
+
+image:
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
+
+commandConfig:
+  command: [ "bin/pdnd-interop-uservice-party-management" ]
+  args: [ "-Dconfig.override_with_env_vars=true" ]
+
+imagePullSecrets:
+  - name: regcred
+
+nameOverride: ""
+fullnameOverride: "pdnd-interop-uservice-party-management"
+
+containers:
+  portNumber: 8088
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  runAsUser: 1001
+  allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+
+service:
+  type: ClusterIP
+  port: 8088
```

Comment:
Please in this case that we have 2 service use something like "appPort" and "appTargetPort"